demo/src/style/app.scss

```diff
@@ -228,7 +228,10 @@ select::-ms-expand {
   width: 100%;
 }
 .width-50 {
-  width: 50%;
+  width: 100%; // Overridden purely for demo purposes.
```

Comment:
Because the actual component has a `width-50` class and is rendered that way within **Manage cases**. If I also stuck a `width-100` class on it, it might break it within **Manage cases** too. This is only within the demo, making it more visually appealing, and doesn't affect any production code so I thought it'd be fine. 🤷‍♂️

Reply:
Unless there's disagreement I'm treating this as resolved (although leaving this open so you can disagree if not)